k9mail/src/main/AndroidManifest.xml

```diff
@@ -415,11 +419,11 @@
             android:grantUriPermissions="true"
             android:multiprocess="true"
             android:readPermission="${applicationId}.permission.READ_ATTACHMENT">
-            
```

Comment:
To avoid accidental unrelated changes such as this one please configure your IDE to not change lines that you haven't modified.